### PR TITLE
Add thewoksoflife.com

### PR DESCRIPTION
**Pull request by jerjou**
For example: https://thewoksoflife.com/whole-wheat-mantou/

### CODE REVIEW

File: parsers/thewoksoflife.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        for r in d['@graph']:
+            if not isinstance(r['@type'], str):
+                continue
+
+            if r['@type'].lower() != 'recipe':
+                continue
+
+            recipe['name'] = r['name']
+            recipe['description'] = r['description']
+            recipe['ingredients'] = r['recipeIngredient']
```
Comment (poundifdef) [inline parsers/thewoksoflife.py:18]: One thing that I notice is that, for some reason, their ingredients have too many parenthesis characters. In the example URL you provide, the JSON lists ingredients as: `1 teaspoon active dry yeast ((3 grams))`

Could you correct for this? Maybe something as simple as replacing `((` with `(`, same for the closing parentheses.

Comment (jerjou) [reply parsers/thewoksoflife.py]: Yeah - I noticed that too. I wonder why it happens.

Anyway, did a simple substitution. Also propogated it to the other schema-based parsers.

**Comment by poundifdef** [comment]
Thanks for making the update! I see that you have updated other websites besides `thewoksoflife.com`, which was the original intention of your PR. Did you observe this error in other recipe websites as well?

**Comment by jerjou** [comment]
Ah - I misinterpreted your comment to mean that _you_ had seen the double-parentheses on the other sites. I did a quick check and it doesn't appear to be commonplace, so I've reverted those other classes.

**Comment by poundifdef** [comment]
This is looking great. Will merge it now. Thank you!

**Pull request merged**